go/vt/vtgate/planbuilder/select.go

```diff
@@ -168,12 +163,12 @@ func (pb *primitiveBuilder) processSelect(sel *sqlparser.Select, reservedVars *s
 	// Check and error if there is any locking function present in select expression.
 	for _, expr := range sel.SelectExprs {
 		if aExpr, ok := expr.(*sqlparser.AliasedExpr); ok && sqlparser.IsLockingFunc(aExpr.Expr) {
-			return vterrors.Errorf(vtrpcpb.Code_UNIMPLEMENTED, "%v allowed only with dual", sqlparser.String(aExpr))
+			return vterrors.VT12001(fmt.Sprintf("%v allowed only with dual", sqlparser.String(aExpr)))
```

Comment:
```suggestion
			return vterrors.VT12001(fmt.Sprintf("%v is allowed only with dual", sqlparser.String(aExpr)))
```

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)